src/index.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
+<html lang="en">
+
+  <head>
+    <meta charset="utf-8">
+    <title>OppiaAngular</title>
```

Comment:
The default title can be `Loading | Oppia`. This title will be overridden as soon as the lazy loaded page module is bootstrapped.